20/alpine/Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM alpine:3.8
 
 ENV OTP_VERSION="20.3.8.10"
+ENV OTP_DOWNLOAD_SHA256="137562e327c1772ce10b44e7d7a3d2bc00b618aee899dd69d2d1e24012854a19"
+ENV REBAR3_VERSION="3.6.1"
+ENV REBAR3_DOWNLOAD_SHA256="40b3c85440f3235c7b149578d0211bdf57d1c66390f888bb771704f8abc71033"
```

Comment:
see my other elixir Dockerfile, the `ENV` instruction support multiple environment in one instruction:

https://github.com/c0b/docker-elixir/blob/master/1.7/Dockerfile

Reply:
and why do you think move the `OTP_DOWNLOAD_SHA256` from a shell variable to an env variable is better?

this depends on the question do we need this SHA256 to be visiable to final users of this image?

this is current image: I leave the `OTP_VERSION` there because I feel it's useful for user's application to detect version by this env variable

```console
$ docker run -it --rm erlang:21-alpine sh
/ # 
/ # env
HOSTNAME=c0956037a62a
SHLVL=1
HOME=/root
OTP_VERSION=21.1.1
TERM=xterm
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PWD=/
/ # 
/ # erl
Erlang/OTP 21 [erts-10.1.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

Eshell V10.1.1  (abort with ^G)
1> os:get_env_var("OTP_VERSION").
"21.1.1"
2> 
```